incubator/istio/values.yaml

```diff
@@ -180,7 +173,7 @@ addons:
     deployment:
       name: prometheus
       annotations:
-        alpha.istio.io/sidecar: ignore
+        sidecar.istio.io/inject: "false"
       image: quay.io/coreos/prometheus
       imageTag: v1.1.1
```

Comment:
1.1.1 sounds ancient. 1.7.2 is current.

Reply:
There is nothing stopping Istio from requiring 1.1.1 if that's what it is tested against.

Reply:
This is what release 0.2.6 calls for

Reply:
OK, I was just wondering.